.github/workflows/ci-cd.yaml

```diff
@@ -25,9 +25,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2.4.0
       - name: Node ${{ matrix.node_version }} - x64 on ${{ matrix.os }}
-        uses: actions/setup-node@v2.4.1
+        uses: actions/setup-node@v2.5.0
         with:
-          node-version: ${{ matrix.node_version }}
```

Comment:
`node_version` variable is now unused. Let's remove the whole matrix L17-24 and replace related `${{ matrix.` variables with their values.